src/pages/shops/ShopLanding.jsx

```diff
@@ -21,8 +26,8 @@ const ShopLanding = () => {
             <div className="shop_textdiv">
               <p className="shop_header1">Shop for charity !</p>
               <p className="shop_header2">
-                Welcome to our shop. Anything you buy from here helps out the
-                charities. We don't take anything from the profit.
+                {`Welcome to our shop. Anything you buy from here helps out the
```

Comment:
Why did you add the `{` ?

Reply:
yes I was getting linting error on ' so I wrapped it inside curly braces

Reply:
Cool, just replace don't with `don&apos;t`

We use the `&apos;` for '

Reply:
I have made the change please check